packages/graph-explorer/src/modules/EntitiesTabular/components/EdgesTabular.tsx

```diff
@@ -20,7 +20,23 @@ import useTranslations from "@/hooks/useTranslations";
 import { recoilDiffSets } from "@/utils/recoilState";
 import { DisplayEdge, useDisplayEdgesInCanvas } from "@/core";
 
-type ToggleEdge = DisplayEdge & { __is_visible: boolean };
+/** Creates the model for the table data */
+function createEdgeForTable(edge: DisplayEdge) {
+  return {
+    id: edge.id,
+    displayTypes: edge.displayTypes,
+    displayName: edge.displayName,
+    sourceDisplayId: edge.source.displayId,
+    sourceDisplayTypes: edge.source.displayTypes,
+    targetDisplayId: edge.target.displayId,
+    targetDisplayTypes: edge.target.displayTypes,
+  };
+}
```

Comment:
Flattening the nested `DisplayEdge` source and vertex info to make the table easier to deal with.